systems/c64.h

```diff
@@ -1032,23 +1063,30 @@ static void _c64_init_key_map(c64_t* sys) {
     }
 
     /* special keys */
-    kbd_register_key(&sys->kbd, 0x20, 4, 7, 0);    /* space */
-    kbd_register_key(&sys->kbd, 0x08, 2, 0, 1);    /* cursor left */
-    kbd_register_key(&sys->kbd, 0x09, 2, 0, 0);    /* cursor right */
-    kbd_register_key(&sys->kbd, 0x0A, 7, 0, 0);    /* cursor down */
-    kbd_register_key(&sys->kbd, 0x0B, 7, 0, 1);    /* cursor up */
-    kbd_register_key(&sys->kbd, 0x01, 0, 0, 0);    /* delete */
-    kbd_register_key(&sys->kbd, 0x0C, 3, 6, 1);    /* clear */
-    kbd_register_key(&sys->kbd, 0x0D, 1, 0, 0);    /* return */
-    kbd_register_key(&sys->kbd, 0x03, 7, 7, 0);    /* stop */
-    kbd_register_key(&sys->kbd, 0xF1, 4, 0, 0);
-    kbd_register_key(&sys->kbd, 0xF2, 4, 0, 1);
-    kbd_register_key(&sys->kbd, 0xF3, 5, 0, 0);
-    kbd_register_key(&sys->kbd, 0xF4, 5, 0, 1);
-    kbd_register_key(&sys->kbd, 0xF5, 6, 0, 0);
-    kbd_register_key(&sys->kbd, 0xF6, 6, 0, 1);
-    kbd_register_key(&sys->kbd, 0xF7, 3, 0, 0);
-    kbd_register_key(&sys->kbd, 0xF8, 3, 0, 1);
+    kbd_register_key(&sys->kbd, C64_KEY_SPACE   , 4, 7, 0);    /* space */
+    kbd_register_key(&sys->kbd, C64_KEY_CSRLEFT , 2, 0, 1);    /* cursor left (shift+cursor right)*/
+    kbd_register_key(&sys->kbd, C64_KEY_CSRRIGHT, 2, 0, 0);    /* cursor right */
+    kbd_register_key(&sys->kbd, C64_KEY_CSRDOWN , 7, 0, 0);    /* cursor down */
+    kbd_register_key(&sys->kbd, C64_KEY_CSRUP   , 7, 0, 1);    /* cursor up (shift+cursor down)*/
+    kbd_register_key(&sys->kbd, C64_KEY_DEL     , 0, 0, 0);    /* delete */
+    kbd_register_key(&sys->kbd, C64_KEY_INST    , 0, 0, 1);    /* inst (shift+del)*/
+    kbd_register_key(&sys->kbd, C64_KEY_HOME    , 3, 6, 0);    /* home */
+    kbd_register_key(&sys->kbd, C64_KEY_CLR     , 3, 6, 1);    /* clear (shift+home) */
+    kbd_register_key(&sys->kbd, C64_KEY_RETURN  , 1, 0, 0);    /* return */
+    kbd_register_key(&sys->kbd, C64_KEY_CTRL    , 2, 7, 0);    /* restore (connected to the NMI line) */
+    kbd_register_key(&sys->kbd, C64_KEY_CBM     , 5, 7, 0);    /* ctrl */
+    kbd_register_key(&sys->kbd, C64_KEY_RESTORE , 0, 8, 0);    /* C= commodore key */
```

Comment:
The comments here don't match the key constants.

Reply:
oops sorry, the result of copy-paste. It's fixed now.